apps/src/applab/applab.js

```diff
@@ -292,14 +307,70 @@ function adjustAppSizeStyles(container) {
 var drawDiv = function () {
   var divApplab = document.getElementById('divApplab');
   divApplab.style.width = Applab.appWidth + "px";
-  divApplab.style.height = Applab.appHeight + "px";
+  divApplab.style.height = Applab.footerlessAppHeight + "px";
   if (Applab.levelHtml === '') {
     // On clear gives us a fresh start, including our default screen.
     designMode.loadDefaultScreen();
     designMode.serializeToLevelHtml();
   }
+
+  if (studioApp.share) {
+    renderFooterInSharedGame();
+  }
 };
 
+function renderFooterInSharedGame() {
+  var divApplab = document.getElementById('divApplab');
+  var footerDiv = document.createElement('div');
+  footerDiv.setAttribute('id', 'footerDiv');
+  divApplab.parentNode.insertBefore(footerDiv, divApplab.nextSibling);
+
+  var menuItems = [
+    {
+      text: applabMsg.makeMyOwnApp(),
+      link: '/projects/applab'
+    },
+    {
+      text: commonMsg.openWorkspace(),
+      link: location.href + '/view'
+    },
+    // Disabled until we do the work to support abuse reporting
+    // {
+    //   text: applabMsg.reportAbuse(),
+    //   link: '#'
+    // },
+    {
+      text: applabMsg.copyright(),
+      link: '#',
+      copyright: true
+    },
+    {
+      text: applabMsg.privacyPolicy(),
+      link: 'https://code.org/privacy'
+    }
+  ];
+  if (!dom.isMobile()) {
```

Comment:
We want to remove the make my own app option `if (dom.isMobile())`, right? This is doing the opposite.